internal/nightfallconfig/nightfall_config.go

```diff
@@ -42,30 +63,26 @@ type Config struct {
 
 // GetNightfallConfigFile loads nightfall config from file, returns default if missing/invalid
 func GetNightfallConfigFile(workspacePath, fileName string, logger logger.Logger) (*NightfallConfigFileStructure, error) {
-	defaultNightfallConfig := &NightfallConfigFileStructure{
-		Detectors:         []*nightfallAPI.Detector{&apiKeyDetector, &cryptoKeyDetector},
-		MaxNumberRoutines: DefaultMaxNumberRoutines,
-	}
 	nightfallConfigFile, err := os.Open(path.Join(workspacePath, fileName))
 	if err != nil {
 		logger.Warning(fmt.Sprintf("Error opening nightfall config: %s", err.Error()))
-		logger.Info(defaultDetectorsInfoMessage)
+		logger.Info(defaultConditionsInfoMessage)
 		return defaultNightfallConfig, nil
 	}
 	defer nightfallConfigFile.Close()
 	byteValue, err := ioutil.ReadAll(nightfallConfigFile)
 	if err != nil {
 		logger.Warning(fmt.Sprintf("Error reading nightfall config: %s", err.Error()))
-		logger.Info(defaultDetectorsInfoMessage)
+		logger.Info(defaultConditionsInfoMessage)
 		return defaultNightfallConfig, nil
 	}
 	var nightfallConfig NightfallConfigFileStructure
 	err = json.Unmarshal(byteValue, &nightfallConfig)
 	if err != nil {
 		return nil, err
 	}
-	if len(nightfallConfig.Detectors) < 1 {
-		return nil, errors.New("Nightfall config file is missing detectors")
+	if len(nightfallConfig.Conditions) < 1 {
```

Comment:
[mark] im not sure, are we planning to accept only conditionSetUUID for these pr scanner

Reply:
+1 I think we should also add a `ConditionSetUUID` field wherever you're changing `Detectors` -> `Conditions`, and update this if condition to:
`if len(nightfallConfig.Conditions) < 1 && nightfallConfig.ConditionSetUUID == nil {`

Reply:
yep good call - i just figured that could be added in a separate PR